Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM node:8.4
+FROM node:10.15.3-alpine
```

Comment:
[SUGGESTION] Should we change this to `node:10-alpine` so it keeps up with the latest versions of node 10?  Worst case `node:10.16-alpine`?

Reply:
@rmm5t agree, have pushed that change.